docs/classic-ui/mockup.md

```diff
@@ -18,16 +18,16 @@ View the [interactive documentation of Mockup](https://plone.github.io/mockup/).
 
 ## Get started
 
-[bobtemplates.plone](https://github.com/plone/bobtemplates.plone) provides [mr.bob](https://mrbob.readthedocs.io/en/latest/) templates to generate packages for Plone projects.
-[plonecli](https://github.com/plone/plonecli) provides a command line client for bobtemplates.plone.
+[`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone) provides [`mr.bob`](https://github.com/collective/mr.bob) templates to generate packages for Plone projects.
+[Plone CLI (`plonecli`)](https://github.com/plone/plonecli) provides a command line client for `bobtemplates.plone`.
 
-Install plonecli into your Python user packages to make it available to all your projects.
+Install {term}`plonecli` into your Python user packages to make it available to all your projects.
 
 ```shell
 pip install plonecli --user
 ```
 
-Create a theme package add-on with {term}`plonecli`.
+Create a theme package add-on with `plonecli`.
 
 ```shell
 plonecli add mockup_pattern
```

Comment:
```suggestion
plonecli create addon project.addon
```

This will create a `project.addon` package which can be installed in your Plone site.

You can `cd` to the project, and add features to that package, such as content-types, behaviors, controlpanels or REST API endpoints.

```shell
cd project.addon
plonecli add content_type
plonecli add behavior
plonecli theme_barceloneta
```

Each of the features asks several questions to create the desired feature as customized as possible.

You can check the full list of available features using the `-l` parameter:

```shell
plonecli -l
```

Reply:
Added in https://github.com/plone/documentation/pull/1734/commits/b0881aef23b621f3998a4a54baf71cd2872bac4b.

Thanks for the review!